src/kixi/hecuba/onyx/components/onyx_job.clj

```diff
@@ -0,0 +1,66 @@
+(ns kixi.hecuba.onyx.components.onyx-job
+  (:require [com.stuartsierra.component :as component]
+            [taoensso.timbre :as timbre]
+            [cheshire.core :as json]))
+
+(defonce state (atom nil))
+(def id (java.util.UUID/randomUUID))
+(def batch-size 10)
+(def batch-timeout 1000)
+
+(def env-config
+  {:zookeeper/address "127.0.0.1:2181"
```

Comment:
bikeshedding 😉  because initial commit, but good idea to read such all things from config files as you know